common/tasks/scheduler.go

```diff
@@ -24,6 +24,7 @@
 
 package tasks
 
+//go:generate go run ../../cmd/tools/mockgen -copyright_file ../../LICENSE -package $GOPACKAGE -source $GOFILE -destination scheduler_mock.go
```

Comment:
This command somehow got lost, and I only noticed after deleting mocks and re-running the generate command. We probably want to preserve it since the mocks are actually used.